tools/lint/lint.py

```diff
@@ -747,8 +747,18 @@ def changed_files(wpt_root):
 
 def lint_paths(kwargs, wpt_root):
     if kwargs.get("paths"):
-        r = os.path.realpath(wpt_root)
-        paths = [os.path.relpath(os.path.realpath(x), r) for x in kwargs["paths"]]
+        #make new list in case there are multiple added directories
+        paths = []
+        for path in kwargs.get("paths"):
+            if os.path.isdir(path):
```

Comment:
I think this  `if` is missing an else clause to just add the path to the list of paths.